tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/vampire.ts

```diff
@@ -43,7 +35,6 @@ const Vampire: Species = {
   },
   lore: [
     "Vampires are unholy beings blessed and cursed with The Thirst. The Thirst requires them to feast on blood to stay alive, and in return it gives them many bonuses. Because of this, Vampires have split into two clans, one that embraces their powers as a blessing and one that rejects it.",
-    "\"I'm not doing the bat trick. It's self deprecating. Okay, fine, but I'm not doing it again!\" - Count Baz, before venting into security for the third time this week.",
```

Comment:
Bring this back but change it to the bridge, it's a funny Jerma joke

Reply:
> Bring this back but change it to the bridge, it's a funny Jerma joke

but they can't do it anymore, since they can't turn into bats anymore

Reply:
Oh that sucks @tralezab we should find a new Jerma joke